app/src/main/java/com/framgia/oleo/screen/home/HomeFragment.kt

```diff
@@ -153,6 +170,10 @@ class HomeFragment : BaseFragment(), BottomNavigationView.OnNavigationItemSelect
         private const val REQUEST_APP_DETAILS = 99
         private const val PERMISSION_REQUEST = 10
         private const val PERMISSION_DENIED = "Permission denied"
+        private const val INDEX_ZERO = 0
```

Comment:
```suggestion
        private const val TAB_MESSAGE = 0
```
sửa tương tự các tab còn lại